cmd/crc/cmd/root.go

```diff
@@ -160,12 +160,7 @@ func newMachine() machine.Client {
 }
 
 func newMachineWithConfig(config crcConfig.Storage) machine.Client {
-	return machine.NewClient(
-		constants.DefaultName,
-		isDebugLog(),
-		network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString()),
-		config.Get(cmdConfig.EnableClusterMonitoring).AsBool(),
-	)
+	return machine.NewClient(constants.DefaultName, network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString()), config.Get(cmdConfig.EnableClusterMonitoring).AsBool())
```

Comment:
almost tempted to write this as
```
    networkMode := network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString())
    enableMonitoring := config.Get(cmdConfig.EnableClusterMonitoring).AsBool()
    machine.NewClient(constants.DefaultName, networkMode, enableMonitoring)
```
for readability.

Reply:
yes done